### PR TITLE
Branch add module

### LINKED ISSUE #35: Refactor model to add module skeletal code

### PR DESCRIPTION
**Pull request by hanstirtaputra**
Added some skeletal code for classes related to our Module feature.

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -49,13 +49,20 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Developer
 * Responsibilities: UI
 
+<<<<<<< HEAD
```
Comment (khoongwk) [inline docs/AboutUs.md:52]: Delete this "<<<<<<< HEAD"?

File: docs/AboutUs.md [hunk 2]
```diff
@@ -49,13 +49,20 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
+### Hans Sebastian Tirtaputra
+
+<img src="images/hansebastian.png" width="200px">
+
+[[github](http://github.com/hansebastian)]
+[[portfolio](team/hansebastian.md)]
+=======
```
Comment (khoongwk) [inline docs/AboutUs.md:59]: Delete this too

**Review by khoongwk — APPROVED** [review]
Nice skeletal classes, LGTM. However, we need to note that @hansebastian 's changes will incorporate Remark command into GrAB3. @RuiFengg @galvinleow @yanlynnnnn

**Comment by RuiFengg** [comment]
Not sure if add remark changes were accidental for this branch but would be useful to have it to serve as the description for tasks :smile_cat:

**Pull request closed without merging**